dispatch/sdk/v1/function.proto

```diff
@@ -62,6 +63,14 @@ message RunRequest {
   // hierarchy tree is created. This field carries the identifier of the
   // root function call in the tree.
   string root_dispatch_id = 6;
+
+  // Time at which the function call was created.
+  //
+  // Note that if the function tail-calls, the timestamp is not reset.
+  google.protobuf.Timestamp created_at = 7;
+
+  // Time at which the function call expires.
+  google.protobuf.Timestamp expires_at = 8;
```

Comment:
I wonder if `expired_at` could be a more future-proof wording? Say we read this value post-expiration; using the present tense would be a little weird. "expired" not as a past tense but a passive form, as in "this call will be expired at" could still be meaningful.

You're the native speaker tho, I will defer to you on that.

Reply:
I'd agree if this were a database table and you were potentially reading historical records. In this case though this a request that should not be observed outside the bounds of these timestamps (with some minor leeway for timestamps out of sync across systems). The vast majority of use cases are going to consume the data within the request-response lifecycle, where the present tense makes more sense.

That said, how about we avoid both and instead use `creation_time` and `expiry_time`?

Reply:
> That said, how about we avoid both and instead use creation_time and expiry_time?

Sounds good 👍

Reply:
Is "expiry" better in English than "expiration"?

Reply:
Expiration seems to be more common in American English. I'll stick with that 🙂